types/deneb/state.yaml

```diff
@@ -0,0 +1,88 @@
+Deneb:
+  BeaconState:
+    type: object
+    description: "The [`BeaconState`](https://github.com/ethereum/consensus-specs/blob/master/specs/deneb/beacon-chain.md#beaconstate) object from the Eth2.0 Deneb spec."
+    properties:
+      genesis_time:
+        $ref: "../primitive.yaml#/Uint64"
+      genesis_validators_root:
+        $ref: "../primitive.yaml#/Root"
+      slot:
+        $ref: "../primitive.yaml#/Uint64"
+      fork:
+        $ref: "../misc.yaml#/Fork"
+      latest_block_header:
+        $ref: "../block.yaml#/BeaconBlockHeader"
+      block_roots:
+        type: array
+        description: "Fixed length of 8192 items"
+        items:
+          $ref: '../primitive.yaml#/Root'
+      state_roots:
+        type: array
+        description: "Fixed length of 8192 items"
+        items:
+          $ref: '../primitive.yaml#/Root'
+      historical_roots:
```

Comment:
Is this now a 0-length array in the spec, given that it has been superseded by `historical_summaries`?  Also, we're missing `historical_summaries` in this definition.

Reply:
It looks like we're missing historical_summaries in the 2.4.0 release from capella, so that's a gap I might raise an issue on...  This is consistent with the Capella definition in master, so we need to fix capella and deneb both.

Reply:
https://github.com/ethereum/consensus-specs/blob/dev/specs/capella/beacon-chain.md#beaconstate
I'm not sure it's a 0 length, but whatever it ends up, it does look like tech debt...

Reply:
ok, was easy enough for capella, brought in change and moved to deneb also.